polaris-react/src/components/VideoThumbnail/VideoThumbnail.tsx

```diff
@@ -73,13 +77,19 @@ export function VideoThumbnail({
   }
 
   const timeStampMarkup = videoLength ? (
-    <p
-      className={classNames(
-        styles.Timestamp,
-        showVideoProgress && styles.withProgress,
-      )}
-    >
-      {secondsToTimestamp(videoLength)}
+    <p className={styles.Timestamp}>
+      <Stack alignment="center" spacing="extraTight">
+        <span className={styles.PlayIcon}>
+          <Icon source={PlayMinor} />
+        </span>
+        <Text
```

Comment:
We have a `Text` component now?! 🤯

Reply:
Yesssssss. Spread the word @mrcthms 🚀 
https://polaris.shopify.com/whats-new/version-10-typography